src/MainMenu.java

```diff
@@ -44,10 +76,12 @@
 
         GridLayout grid = new GridLayout(2, 0, 10, 20);
 
-        gameSelectionPanel.setLayout(grid);
+        //Added Absolute Positioning
+        gameSelectionPanel.setLayout(null);
```

Comment:
The whole layout seems that is not justified in the middle. Try and make the layout and text be in the middle of the screen.

![image](https://user-images.githubusercontent.com/36922092/97861086-e0e6cd00-1d0b-11eb-91cd-e0ae11ba6565.png)


Also please notice that the screen is now too empty, try and make the components a bit bigger

Reply:
I guess you should test it with different screen sizes as well. Also, make sure to fix the other things mentioned.